textgrad/engine/__init__.py

```diff
@@ -1,11 +1,19 @@
 from .base import EngineLM, CachedEngine
-
+from transformers import AutoConfig
 __ENGINE_NAME_SHORTCUTS__ = {
     "opus": "claude-3-opus-20240229",
     "haiku": "claude-3-haiku-20240307",
     "sonnet": "claude-3-sonnet-20240229",
     "together-llama-3-70b": "together-meta-llama/Llama-3-70b-chat-hf",
 }
+def is_in_hf(engine_name):
+    exists_on_hf = True
+    try:
+        tok = AutoConfig.from_pretrained(engine_name)
+    except:
```

Comment:
can we do the exception handling here based on the right type of error? e.g. can we ensure it truly does not exist in HF, and it's not some versioning error, or weird things like we need to add trust_remote_code or something?